reports/models.py

```diff
@@ -216,3 +217,46 @@ class UploadLevelTotals(AbstractTotals):
 
     class Meta:
         db_table = "reports_uploadleveltotals"
+
+
+class Test(models.Model):
+    id = models.TextField(primary_key=True)
+
+    external_id = models.UUIDField(default=uuid.uuid4, editable=False)
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    repository = models.ForeignKey(
+        "core.Repository",
+        db_column="repoid",
+        related_name="tests",
+        on_delete=models.CASCADE,
+    )
+    name = models.TextField()
+    testsuite = models.TextField()
+    env = models.TextField(default="")
```

Comment:
TODO: if you're gonna have a default, just set null=true, see this for reference https://www.notion.so/sentry/Database-Tips-and-Tricks-76df725ded264b2a8154c960d7ef3869?pvs=4#753bec6f64e0462c8f4d9379e79af9a2